packages/atomic/src/components/insight/atomic-insight-full-search-button/atomic-insight-full-search-button.tsx

```diff
@@ -0,0 +1,30 @@
+import {Component, h, Prop} from '@stencil/core';
+import ArrowFull from '../../../images/arrow-full.svg';
+import {IconButton} from '../../common/iconButton';
+
+/**
+ * @internal
+ */
+@Component({
+  tag: 'atomic-insight-full-search-button',
+  styleUrl: 'atomic-insight-full-search-button.pcss',
+  shadow: true,
+})
+export class AtomicInsightFullSearchButton {
+  @Prop({mutable: true}) public clickCallback: () => void = () => {};
```

Comment:
Will this always just be a redirection to another page, or are there some instances where this will basically just be an expand button in a single-page application?

If it's always just a redirection, then I think that we should use the `a` tag instead of the `button` tag, which allows people to right click and press "Open in new tab", but also conveys to accessibility tools that clicking it will exit the current page.

Reply:
As of now the only usage will be in the builder preview to reflect a real functioning button rendered by quantic in salesforce so it will be no-op. When we start providing Atomic insight panels outside of the builder then it's still unclear what this button will be used for or if it will be allowed at all.

Reply:
I would propose then just removing that property, it does not seem meaningful to add one yet ?